integration-tests/pkg/assert/assert.go

```diff
@@ -1,13 +1,67 @@
 package assert
 
 import (
+	"encoding/json"
 	"fmt"
+	"strings"
 	"testing"
+	"time"
 
 	"github.com/davecgh/go-spew/spew"
 	"github.com/stretchr/testify/assert"
+
+	"github.com/stackrox/collector/integration-tests/pkg/collector"
+	"github.com/stackrox/collector/integration-tests/pkg/log"
+	"github.com/stackrox/collector/integration-tests/pkg/types"
+)
+
+var (
+	runtimeConfigErrorMsg = "Runtime configuration was not updated"
 )
 
+func AssertExternalIps(t *testing.T, enable bool, collectorIP string) {
```

Comment:
[nit] should probably be `enabled` rather than `enable` (the latter implies we're turning it on)

Reply:
Done